.devcontainer/scripts/usr/local/bin/postCreateCommand.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 b-data GmbH.
+# Distributed under the terms of the MIT License.
+
+set -e
+
+# Change ownership of the stack folder
+sudo chown "$(id -u)":"$(id -g)" /workspaces/stack
```

Comment:
@benz0li, for my benefit, pleae can you explain what this does and why it is needed/desirable.

Reply:
@mpilgrem Fixes https://github.com/commercialhaskell/stack/issues/6620